### PR TITLE
SE Information

### PR DESCRIPTION
**Pull request by hshahwan**
work on SE Information page 
* Business info.
* Location and places.
* Progress Trackers.
* Turn Over.
* upload logo.
* social impact.
* Health and safety policy.

### CODE REVIEW

File: client/src/Components/App.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ import HomePage from './Pages/HomePage';
 import ResultPage from './Pages/ResultPage';
 import Signup from './Pages/Signup';
 import Header from './CommonComponents/Header';
-import Login from './CommonComponents/Login'
+import Login from './CommonComponents/Login';
+import SEInofrmation from './Pages/SEInformation';
```
Comment (sultanassi95) [inline client/src/Components/App.js]: `SEInofrmation` ==> `SEInformation`

File: client/src/Components/App.js [hunk 2]
```diff
@@ -37,6 +40,7 @@ class App extends Component {
               <Route path='/contract/find'  render={()=><ResultPage SEId={this.state.businessId} />} exact  />
               <Route path='/signup' component={Signup} exact />
               <Route path='/login' component={Login} exact />
+              <Route path='/seinofrmation' component={SEInofrmation} exact />
```
Comment (sultanassi95) [inline client/src/Components/App.js]: `SEInofrmation` ==> `SEInformation`

File: client/src/Components/Pages/HomePage/HeroSection/style.css [hunk 3]
```diff
@@ -26,12 +26,12 @@
...
 .findMore {
-  width: 40%;
+  width: 60%;
   margin: 0 auto;
   background: #2D918E;
-  height: 20px;
+  height: 40px;
   text-align: center;
-  padding: 15px;
+  /* padding: 15px; */
```
Comment (sultanassi95) [inline client/src/Components/Pages/HomePage/HeroSection/style.css]: if u r not using it, delete it please :pray:

File: client/src/Components/App.js (hunk 1 above)
Comment (hshahwan) [reply client/src/Components/App.js]: oh, will change it :laughing:

**Pull request closed without merging**